pkgs/development/python-modules/mozjpeg_lossless_optimization/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  buildPythonPackage,
+  mozjpeg,
+  pytestCheckHook,
+  setuptools,
+  cmake,
+}:
+buildPythonPackage rec {
+  pname = "mozjpeg_lossless_optimization";
+  version = "1.1.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "wanadev";
+    repo = "mozjpeg-lossless-optimization";
+    # https://github.com/NixOS/nixpkgs/issues/26302
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-OKNt9XtfZ6hhRJN1Asn1T2dVjyXKQAsnFvXKYnrRZ98=";
+    fetchSubmodules = true;
+  };
+
+  # This package needs cmake, but it is not the default builder
+  dontUseCmakeConfigure = true;
+
+  buildInputs = [ mozjpeg ];
+  nativeBuildInputs = [ cmake ];
+  propagatedBuildInputs = [ python3Packages.cffi ];
+
+  # https://github.com/NixOS/nixpkgs/issues/255262
+  preCheck = ''
+    rm -r mozjpeg_lossless_optimization
+  '';
+
+  build-system = [ setuptools ];
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = with lib; {
+    description = "Python library to optimize JPEGs losslessly using MozJPEG";
+    homepage = "https://github.com/wanadev/mozjpeg-lossless-optimization";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ adfaure ];
+  };
```

You can remove `with lib` here. Not a blocker. See https://github.com/NixOS/nixpkgs/issues/208242.

```suggestion
  meta = {
    description = "Python library to optimize JPEGs losslessly using MozJPEG";
    homepage = "https://github.com/wanadev/mozjpeg-lossless-optimization";
    license = lib.licenses.bsd3;
    maintainers = with lib.maintainers; [ adfaure ];
  };
```